apps/champions/lib/champions/battle/simulator.ex

```diff
@@ -636,6 +666,16 @@ defmodule Champions.Battle.Simulator do
     Enum.take(sorted_units, amount)
   end
 
+  defp filter_and_choose_units_by_stat(units, stat, count, caster, target_allies, order) do
```

Comment:
I'd rename this to `choose_units_by_stat_and_team`, the "filter" word seems confusing

Reply:
I like it!